src/modules/Channel/components/MessageFeedbackModal/hooks/useKeyDownV2.ts

```diff
@@ -0,0 +1,46 @@
+import { useCallback, useLayoutEffect } from 'react';
+
+type KeyDownCallbackMap = Record<string, (event: React.KeyboardEvent<HTMLDivElement>) => void>;
+
+interface UseKeyDownV2Props {
+  ref: React.RefObject<HTMLDivElement>;
+  keyDownCallbackMap: KeyDownCallbackMap;
+}
+
+export function useKeyDownV2(props: UseKeyDownV2Props): React.KeyboardEventHandler<HTMLDivElement> {
```

Comment:
Q. What's the reason it has V2 suffix?

Reply:
useKeyDown.ts already exists but the old one is not compatible and this new one is the better version.

Reply:
Yeah I saw we have useKeyDown already. But it seems they are not being replaceable but being used in separate places.

Reply:
Yes their goals are the same but V2 is a better one. From now on we should use V2 instead. We can remove V1 later.

Reply:
@AhyoungRyu I refactored `useKeyDown.ts` !